git_machete/exceptions.py

```diff
@@ -4,7 +4,7 @@
 
 
 class MacheteException(Exception):
-    def __init__(self, msg: str, apply_fmt: bool = True, verbose: bool = True) -> None:
+    def __init__(self, msg: str, apply_fmt: bool = True, verbose: bool = False) -> None:
```

Comment:
Hmm can we go step further and just remove this param (and the associated `if` below)?